### PR TITLE
feat: extended calculations with dollar value

### PR DESCRIPTION
**Pull request by yusufseyrek**
* Using `umars` for `emission_per_second`. 
* Adding market returns on top of annual incentives when calculating APY
* Updated README

### CODE REVIEW

File: functions.ts [hunk 1]
```diff
@@ -14,15 +19,71 @@ export const getMarketLiquidityAmount = async (
...
+  const assetOut = findAssetByTokenDenom(
+    pool.pool_assets,
+    ASSET.OSMO.denom
+  ) as PoolAsset;
+
+  const numerator = new BigNumber(assetIn.token.amount).div(assetIn.weight);
+  const denominator = new BigNumber(assetOut.token.amount).div(assetOut.weight);
+
+  const ratio = numerator.dividedBy(denominator);
+  const scale = new BigNumber(1).dividedBy(new BigNumber(1).minus(swapFee));
```
Comment (dancreee) [inline functions.ts:83]: not sure we need to account for swap fee in oracle price

**Review by dancreee — APPROVED** [review]

**Comment by dancreee** [comment]
This looks correct to me, @piobab could you also take a look through to get a second pair of eyes on this logic

File: functions.ts (hunk 1 above)
Comment (yusufseyrek) [reply functions.ts:83]: Probably not. I just use the standard math for calculating spot price from a pool. I think price sources like CoinGecko are not including swapFee in to equation. Will remove it

**Review by piobab — APPROVED** [review]
Calculations LGTM

**Pull request merged**